apps/cnquery/cmd/root.go

```diff
@@ -196,3 +204,103 @@ func GenerateMarkdown(dir string) error {
 
 	return nil
 }
+
+func getCobraScanConfig(cmd *cobra.Command, runtime *providers.Runtime, cliRes *plugin.ParseCLIRes) (*scanConfig, error) {
```

Comment:
this function can return an error but it seems like we are doing `os.Exit` and `log.Fatal()` instead. I think it's better to return an error and let the caller of the function decide what to do with that error

Reply:
+1 good call. I'll adjust that

Reply:
Code updated.